docs/en/latest/plugins/google-logging.md

```diff
@@ -0,0 +1,155 @@
+---
+title: google-logging
```

Comment:
Not sure if `google-cloud-logging-service` would be better 😄

Reply:
Maybe `google-cloud-logging` is enough to identify the product?

Reply:
@juzhiyuan I think `google-cloud-logging` is more appropriate, what do you think?

Reply:
good!